odoo/api.py

```diff
@@ -766,6 +766,45 @@ def _lang(self):
             lang = '_' + lang
         return lang
 
+    def _(self, source: str | LazyGettext, *args, _mod='', **kwargs) -> str:
+        """Translate the term using current environment's language.
+
+        Usage:
+
+        ```
+        self.env._("hello world")  # dynamically get module name
+        self.env._("hello %s", "test", _mod=__name__)
+        self.env._(LAZY_TRANSLATION)
+        ```
+
+        :param source: String to translate or lazy translation
+        :param _mod: The module name (use __name__)
+                     otherwise resolved dynamically
+        :param ...: args or kwargs for templating
+        :return: The transalted string
+        """
+        module = _mod
+        lang = self.lang or self.user.lang or 'en_US'
```

Comment:
Not sure falling back on `self.user.lang` is a good idea from a performance point of view.  Moreover, we don't use that fallback for data translations, why would we use it for code translations?  IMO for translations to be effective, you must set up the context properly.

Reply:
For consistency, I will remove it.
It was the equivalent of "Last resort: attempt to guess the language of the user".